.github/workflows/pr_build.yml

```diff
@@ -13,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java-version: [ 8, 11, 16 ]
+        java-version: [ 8, 11, 17 ]
```

Comment:
Shall we also include 21, as it's the latest LTS version?

Reply:
Added 21 for both ubuntu and macos.